openapi.yaml

```diff
@@ -58,26 +54,30 @@ paths:
               schema:
                 $ref: "#/components/schemas/Problem"
 
-  /validate:
+  /parse:
     post:
-      summary: Validate the given AsyncAPI document.
+      summary: Parse the given AsyncAPI document.
       operationId: validate
       tags:
-        - validate
-        - validator
+        - parse
+        - parser
```

Comment:
Should we use the `parser` tag on the `validate` path too? It's just that the functionality of both endpoints is very similar, and **conceptually** the API resource we are dealing with is the Parser. Then there are two operations on that resource: validate and parse.
Maybe not only use the same tags but also change the path to show this relationship: `parser/validate` and `parser/parse`... although the second one is weird 😅, leading me to think that a user using this might not understand the use case at first.

@magicmatatjahu and @smoya wdyt?

Reply:
Yeah, we should use `parser` tag in both paths :) I don't want to change the parser path - validation is a part of parsing, it's the first stage, and parsing itself is validation + custom operations like custom schema parsing etc. The very idea of `/parse' path comes from the fact that someone would like to get a parsed AsyncAPI document, not just a validated one - that's why both paths are important from the point of view of the offered functionality.

Reply:
> parsing itself is validation + custom operations like custom schema parsing etc

Should this be visible right in the `openapi.yml` or some documentation? Maybe it's documented on the parser library, but does a user with no prior knowledge of how the parser works, know when to call /validate or /parse?
I'm fine with the paths as it is 👍 , just wondering if the difference could be documented.

Reply:
Honestly, we do not have it explained, probably nowhere. For example for me it was obvious even before working for AsyncAPI, but I agree that we should describe it. I will describe it in the next opened PR, ok? :)